iphone/iphone/Titanium.plist

```diff
@@ -57,5 +57,23 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+    <key>UIApplicationShortcutItems</key>
```

Comment:
Will be ignored on non-supported devices and improves KitchenSink testing, so its ok to leave it there.